package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -380,10 +595,10 @@ def __init__(self, universe, selection1='protein',
             ``True`` so as to filter out water not residing between the two
             selections. [``True``]
         filter_first : bool (optional)
-            Filter the water selection to only include water within 3 *
-            `distance` away from `both` selection 1 and selection 2.
+            Filter the water selection to only include water within 4Å + `order` *
```

Comment:
There's a space between number and unit: `4 Å`

Reply:
Added.